Issue caused by new version of cfn-resolver-lib

It looks like V 1.1.8 of [cfn-resolver-lib](https://github.com/robessog/cfn-resolver-lib) is causing some issues. I was able to narrow it down to that package/update, but haven't yet uncovered what's actually breaking it. I'm receiving the following error now whenever I try to generate the environment variables:

```
TypeError: Cannot convert undefined or null to object
    at Function.entries (<anonymous>)
    at getParameterDefaults (/home/dane/improovy/ImproovyAWS/services/transcripts/node_modules/cfn-resolver-lib/src/parameterHelper.js:6:54)
    at NodeEvaluator.evaluateNodes (/home/dane/improovy/ImproovyAWS/services/transcripts/node_modules/cfn-resolver-lib/src/nodeEvaluator.js:17:34)
    at /home/dane/improovy/ImproovyAWS/services/transcripts/node_modules/serverless-export-env/src/lib/resolveCloudFormationEnvVariables.js:98:30
    at tryCatcher (/home/dane/improovy/ImproovyAWS/services/transcripts/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/home/dane/improovy/ImproovyAWS/services/transcripts/node_modules/bluebird/js/release/promise.js:544:35)
    at Promise._settlePromise (/home/dane/improovy/ImproovyAWS/services/transcripts/node_modules/bluebird/js/release/promise.js:604:18)
    at Promise._settlePromise0 (/home/dane/improovy/ImproovyAWS/services/transcripts/node_modules/bluebird/js/release/promise.js:649:10)
    at Promise._settlePromises (/home/dane/improovy/ImproovyAWS/services/transcripts/node_modules/bluebird/js/release/promise.js:729:18)
    at Promise._fulfill (/home/dane/improovy/ImproovyAWS/services/transcripts/node_modules/bluebird/js/release/promise.js:673:18)
    at PromiseArray._resolve (/home/dane/improovy/ImproovyAWS/services/transcripts/node_modules/bluebird/js/release/promise_array.js:127:19)
    at PromiseArray._promiseFulfilled (/home/dane/improovy/ImproovyAWS/services/transcripts/node_modules/bluebird/js/release/promise_array.js:145:14)
    at PromiseArray._iterate (/home/dane/improovy/ImproovyAWS/services/transcripts/node_modules/bluebird/js/release/promise_array.js:115:31)
    at PromiseArray.init [as _init] (/home/dane/improovy/ImproovyAWS/services/transcripts/node_modules/bluebird/js/release/promise_array.js:79:10)
    at Promise._settlePromise (/home/dane/improovy/ImproovyAWS/services/transcripts/node_modules/bluebird/js/release/promise.js:601:21)
    at Promise._settlePromise0 (/home/dane/improovy/ImproovyAWS/services/transcripts/node_modules/bluebird/js/release/promise.js:649:10)
    at Promise._settlePromises (/home/dane/improovy/ImproovyAWS/services/transcripts/node_modules/bluebird/js/release/promise.js:729:18)
    at Promise._fulfill (/home/dane/improovy/ImproovyAWS/services/transcripts/node_modules/bluebird/js/release/promise.js:673:18)
    at PromiseArray._resolve (/home/dane/improovy/ImproovyAWS/services/transcripts/node_modules/bluebird/js/release/promise_array.js:127:19)
    at PromiseArray._promiseFulfilled (/home/dane/improovy/ImproovyAWS/services/transcripts/node_modules/bluebird/js/release/promise_array.js:145:14)
    at Promise._settlePromise (/home/dane/improovy/ImproovyAWS/services/transcripts/node_modules/bluebird/js/release/promise.js:609:26)
    at Promise._settlePromise0 (/home/dane/improovy/ImproovyAWS/services/transcripts/node_modules/bluebird/js/release/promise.js:649:10)
    at Promise._settlePromises (/home/dane/improovy/ImproovyAWS/services/transcripts/node_modules/bluebird/js/release/promise.js:729:18)
    at _drainQueueStep (/home/dane/improovy/ImproovyAWS/services/transcripts/node_modules/bluebird/js/release/async.js:93:12)
    at _drainQueue (/home/dane/improovy/ImproovyAWS/services/transcripts/node_modules/bluebird/js/release/async.js:86:9)
    at Async._drainQueues (/home/dane/improovy/ImproovyAWS/services/transcripts/node_modules/bluebird/js/release/async.js:102:5)
    at Immediate.Async.drainQueues [as _onImmediate] (/home/dane/improovy/ImproovyAWS/services/transcripts/node_modules/bluebird/js/release/async.js:15:14)
    at processImmediate (internal/timers.js:464:21)
```

Same problem here. Can't 1.4.4 with serverless v3 because of dependency restrictions:
```
npm ERR! peer serverless@"1.x || 2.x" from serverless-export-env@1.4.4
npm ERR! node_modules/serverless-export-env
npm ERR!   serverless-export-env@"1.4.4" from the root project
```

--- a/src/lib/resolveCloudFormationEnvVariables.js
+++ b/src/lib/resolveCloudFormationEnvVariables.js
@@ -48,7 +48,7 @@ function resolveGetAtt(refs, resource) {
 
 function resolveResources(AWS, stack, resources, exports, toBeResolved, maps) {
   return BbPromise.all([AWS.getRegion(), AWS.getAccountId()]).spread((region, accountId) => {
-    const node = _.assign({}, { toBeResolved });
+    const node = _.assign({}, { toBeResolved, Parameters: [] });
     const refResolvers = _.merge(
       {
         "AWS::Partition": "aws", // FIXME How to determine correct value?

